Incidents/INCIDENT_PLAYBOOK.md

```diff
@@ -0,0 +1,86 @@
+# Incident Playbook
+
+This is our "playbook" to respond to incidents, such as [PagerDuty alerts](https://www.pagerduty.com).
+
+First of all, **take a deep breath!** :relieved: It can be stressful to get an alert, but we're all in this together. As a first responder, your job is to figure out whether this is a "real" emergency or not. If it is (or you're unsure), don't hesitate to hit the **Escalate** button to summon help! We've got this!
+
+### First-Responder Checklist
+
+1. If you have access to a computer to troubleshoot with, **acknowledge** the incident to let others know you're on it.
```

Comment:
I can see scenarios where we can determine that everything is fine from a phone (hitting the site, re-running ghost inspector). If that's the case, should we only acknowledge once we've determined that nothing is wrong?

Reply:
Yeah, perhaps "computer" is a little too prescriptive. I've totally sanity-checked alerts from my phone before. Would "If you have access to a device to troubleshoot with" sound better? Or "If you have time to troubleshoot this problem"?

Either way though, you'd "acknowledge" as soon as you're looking into things (since that stops the automatic escalation) & "resolve" once you've determined there's no issue.

Reply:
What about "if you are able to troubleshoot this problem now (either on your phone or your computer)"?

That way if someone only has their phone on them they can try to resolve it (they'll be able to figure out if it's an emergency or not I'd think) and likely _can_ resolve it if nothing is wrong, or they can escalate it once they've done as much as they can from their phone.

Reply:
Yeah, good call! 👍